src/Symfony/Component/Form/FormConfigBuilder.php

```diff
@@ -850,6 +850,28 @@ public function setRequestHandler(RequestHandlerInterface $requestHandler)
     }
 
     /**
+     * Adds a request handler to a chain of request handlers.
+     *
+     * @param ChainableRequestHandlerInterface $requestHandler
+     *
+     * @return self The configuration object
+     */
+    public function addRequestHandler(ChainableRequestHandlerInterface $requestHandler)
```

Comment:
I don't think this method is needed. We should explicitly set the RequestHandlerChain instead.